common/build/eslint-config-fluid/printed-configs/default.json

```diff
@@ -276,18 +315,21 @@
             }
         ],
         "@typescript-eslint/require-await": [
-            "off"
+            "error"
         ],
         "@typescript-eslint/restrict-plus-operands": [
             "error"
         ],
         "@typescript-eslint/restrict-template-expressions": [
-            "off"
+            "error"
```

Comment:
i'd prefer this one to stay off, but i also love coercing random garbage into strings this way.  at the very least we should use the options to allow booleans and probably nullish?

Reply:
I guess it depends how we use template strings. If we typically use them for logging or something, maybe it's ok. I think this is right on the edge of being too pedantic, so I wouldn't be sad if we disabled it.